### PR TITLE
Support for portuguese [#371]

### PR DESCRIPTION
**Pull request by jmcrfp**
## Status
**READY**

## Migrations
NO

## Description
Support for portuguese

### CODE REVIEW

**Comment by mcpierce** [comment]
Will you be pushing a follow up PR for the translations? I see that the backend-status.json file isn't translated in this PR.

File: comixed-frontend/src/assets/i18n/pt/library.json [hunk 1]
```diff
@@ -169,378 +169,378 @@
...
-    "title": "ComiXed: Story Arcs ({count})"
+    "title": "ComiXed: Arcos da historias ({count})"
   },
   "story-arc-details-page": {
-    "title": "ComiXed: Story Arc - {name} ({count, plural, =1{One Comic} other{# Comics}})"
+    "title": "ComiXed: Arco da historia - {name} ({count, plural, =1{Um comic} other{# comics}})"
   },
   "duplicates-page": {
-    "title": "{pageCount} Duplicate {pageCount, plural, =1{Page} other{Pages}}",
+    "title": "{pageCount} {pageCount, plural, =1{Pagina} other{Paginas} duplicadas}",
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/pt/library.json]: The final curly brace on this entry is out of place. It should come after Paginas as follows:
```
"title": "{pageCount} {pageCount, plural, =1{Pagina} other{Paginas}} duplicadas",
```

File: comixed-frontend/src/assets/i18n/pt/app.json [hunk 2]
```diff
@@ -316,136 +316,136 @@
...
     "errors": {
-      "form-field-required": "{name} is required."
+      "form-field-required": "{name} é necessário."
     },
     "tooltip": {
-      "unprocessed-comic": "This comic is scheduled for processing.",
-      "deleted-comic": "This comic is marked for deletion."
+      "unprocessed-comic": "Este comic está agendado para ser processado.",
+      "deleted-comic": "Este comic está selecionado para ser apagado."
     }
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/pt/app.json:437]: I found a missing translation from #376 that wasn't added to this file. Can you add the following to the block below tooltip and translate it, please?

```
    "confirmation": {
      "logout": {
        "header": "Logout",
        "message": "Are you sure you want to log out?"
      }
    }
```

**Review by mcpierce — CHANGES_REQUESTED** [review]
All in all looking good. Just one small fix, an addition and one other file to translate and we can merge this.

File: comixed-frontend/src/assets/i18n/pt/library.json (hunk 1 above)
Comment (jmcrfp) [reply comixed-frontend/src/assets/i18n/pt/library.json]: Updated

File: comixed-frontend/src/assets/i18n/pt/app.json (hunk 2 above)
Comment (jmcrfp) [reply comixed-frontend/src/assets/i18n/pt/app.json:437]: Added to the file

**Comment by jmcrfp** [comment]
> Will you be pushing a follow up PR for the translations? I see that the backend-status.json file isn't translated in this PR.

The file was translated. I thought I translated everything. Did I miss something?

**Comment by mcpierce** [comment]
> > Will you be pushing a follow up PR for the translations? I see that the backend-status.json file isn't translated in this PR.
> 
> The file was translated. I thought I translated everything. Did I miss something?

Hrm, I had thought it was still english. But it seems i was mistaken. The issue is that, in the component itself, the i18n entry name was changed so it differs from the files now.

**Review by mcpierce — APPROVED** [review]

**Pull request merged**